### PR TITLE
Register Subdomain: ahad

### PR DESCRIPTION
**Pull request by AhadWeb3**
Added the domain: ahad

### CODE REVIEW

File: domains/ahad.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "AhadWeb3",
+        "email": "ahadweb3.0@gmail.com"
+      },
+
+      "record": {
+        "URL": "ahad"
```
Comment (WilliamDavidHarrison) [inline domains/ahad.json:9]: Invalid URL

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/ahad.json (hunk 1 above)
Comment (AhadWeb3) [reply domains/ahad.json:9]: https://ahadweb3.github.io/Ahad/

Hey!
will this work?

**Comment by AhadWeb3** [comment]
https://ahadweb3.github.io/Ahad/

Hey!
will this work?

**Comment by WilliamDavidHarrison** [comment]
> https://ahadweb3.github.io/Ahad/
> 
> Hey! will this work?

Sorry you need more content on your website.

**Pull request closed without merging**